### DISCUSSION TITLE
Install your through conda?

### DISCUSSION THREAD

**Discussion by cherryng** [thepetabyteproject/your#114] [Ideas]
Hi! Is it possible to have `your` installable through `conda`? I tried installing `your` using `pip` but it is giving me some circular import error.

**Comment by ccalvocm** [comment]
Greetings @cherryng ,

your is not available through conda, nevertheless I'll guide you on how to setup your:

```
conda create -n your python=3.9
conda activate your
pip install your
```

![your](https://github.com/thepetabyteproject/your/assets/52244870/0ec4244f-e1e6-47d5-a112-8ee8bc5d4677)


Kind regards

**Reply by cherryng** [reply]
Thanks, Carlo. I have been able to install `your` alone using `pip`. but the problem arises when I want to have `fetch` in the same environment. The only way I manage to cleanly install `fetch` is using `conda install fetch`. if I combine that with `pip install your`, then the paths are all messed up and gave me circular import error as I briefly mentioned in the initial post...

to simply my question: basically I would like to have `your` and `fetch` in the same environment. is that possible?

**Comment by devanshkv** [comment]
I am working on putting putting it on conda-forge.
